pkg/locale/localizer.go

```diff
@@ -57,6 +57,8 @@ const (
 	FeaturePrerequisiteExists                = "FeaturePrerequisiteExists"
 	FeatureTargetExists                      = "FeatureTargetExists"
 	FeatureRuleExists                        = "FeatureRuleExists"
+	AutoOpsDatetimeClauseExists              = "AutoOpsDatetimeClauseExists"
```

Comment:
You can shorten it by renaming it `AutoOpsHasDatetime` or `AutoOpsHasDatetimeSet`.
Same for the other var names.